webapp-django/crashstats/supersearch/models.py

```diff
@@ -216,14 +232,15 @@ def get(self, **kwargs):
 
 class SuperSearchFields(ESSocorroMiddleware):
 
-    implementation = super_search_fields.SuperSearchFields
+    # Read it in once as a class attribute since it'll never change
+    # unless the .json file on disk changes and if that happens you
+    # will have reloaded the Python process.
+    _fields = SuperSearchFieldsWithoutConfig().get()
```

Comment:
This here is a class attribute, only instanced once. That means if anything modifies it (namely, the ``search_fields`` view in ``supersearch``), every subsequent call of ``get`` will receive the modified version.